weave/scorers/base_scorer.py

```diff
@@ -45,7 +46,13 @@ def _validate_scorer_signature(scorer: Union[Callable, Op, Scorer]) -> bool:
         params = inspect.signature(scorer).parameters
     if "output" in params and "model_output" in params:
         raise ValueError(
-            "Both 'output' and 'model_output' cannot be in the scorer signature; prefer just using `output`."
+            textwrap.dedent(
+                """
+                Both `output` and `model_output` cannot be in the scorer signature at the same time.
```

Comment:
This reads a little awkward to me, maybe move "Both" to "cannot both be" or say something like, "The scorer signature cannot include both output and model_output at the same time."